client/client.go

```diff
@@ -43,11 +58,11 @@ type DraupnirClient interface {
 }
 
 func (c Client) AuthorizationHeader() string {
-	return fmt.Sprintf("Bearer %s", c.Token.RefreshToken)
+	return fmt.Sprintf("Bearer %s", c.token.RefreshToken)
 }
 
-func (c Client) get(url string) (*http.Response, error) {
-	req, err := http.NewRequest(http.MethodGet, url, strings.NewReader(""))
+func (c Client) get(path string) (*http.Response, error) {
+	req, err := http.NewRequest(http.MethodGet, c.url+path, strings.NewReader(""))
```

Comment:
I would expect this to be something like
```go
c.client.Get(c.url+path)
```

Reply:
Ah absolutely. That's my bad, of course this still worse.

Reply:
Cool - the rest looks good; can you ping me when it's ready for a final review?